internal/controller/broom_controller_test.go

```diff
@@ -79,6 +153,38 @@ var _ = Describe("Broom Controller", func() {
 			Expect(err).NotTo(HaveOccurred())
 			// TODO(user): Add more specific assertions depending on your controller's reconciliation logic.
 			// Example: If you expect a certain status condition after reconciliation, verify it here.
+
+			By("Checking if the CronJob is targeted correctly")
+			cronJob := &batchv1.CronJob{}
+			Expect(k8sClient.Get(ctx, types.NamespacedName{Name: cronJobName, Namespace: cronJobNamespace}, cronJob)).To(Succeed())
+
+			target := aiv1alpha1.BroomTarget{
+				Namespace: cronJobNamespace,
+				Name:      cronJobName,
+				Labels: map[string]string{
+					"m3.com/use-broom": "true",
+				},
+			}
+			res := isTargeted(*cronJob, target)
+			Expect(res).To(BeTrue(), "Expected CronJob to be targeted but it is not")
+
+			emptyTarget := aiv1alpha1.BroomTarget{}
+			res = isTargeted(*cronJob, emptyTarget)
```

Comment:
I think ginkgo is for E2E test. So, it is preferable to simple go test for isTargeted.

Reply:
That's right. I will add tests in other PRs.